contrib/examples/walkthrough/README.md

```diff
@@ -1,4 +1,4 @@
 ## Walkthrough Resources
 
 This directory contains API resources for use with the [demo
-walkthrough](../../../docs/walkthrough.md).
+walkthrough](../../../docs/introduction.md).
```

Comment:
I think linking to the introduction doc here is a bit misleading; the new structure of the docs has the core of the walkthrough separated to its own doc(s), with the intro being much larger in scope. I would instead link to the 1.7 walkthrough doc (since the 1.6 one is deprecated, it's unambiguous).

Reply:
@kibbles-n-bytes I updated the language around the link. I want people to go to the intro doc so they can choose whether they want to install or do the demo.